setup.py

```diff
@@ -1,7 +1,12 @@
-from setuptools import setup, find_packages
+import sys
 from codecs import open
 from os import path
 
```

Comment:
Can this extra newline be removed?

Reply:
Imports sorted corresponding to PEP8: https://peps.python.org/pep-0008/#imports

> Imports should be grouped in the following order:
> 
> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
> You should put a blank line between each group of imports.